.github/ISSUE_TEMPLATE/3_discussion.md

```diff
@@ -3,3 +3,5 @@ name: 💬 Ask a question or open a discussion
 about: ———
 labels: under discussion
 ---
+
+<!-- If you wonder why something isn't working, "report a bug" instead of using this template. -->
```

Comment:
People have been doing this…

- https://github.com/refined-github/refined-github/issues/7881
- https://github.com/refined-github/refined-github/issues/7887
- https://github.com/refined-github/refined-github/issues/7370